vtex-apps/product-summary
# Fix props passed to ProductSummaryList

#### What problem is this solving?
It was not possible to change the query being made by ProductSummaryList. This happened because there was a component in the tree that was not passing the props down to the component that did the query.

#### How should this be manually tested?

You can see here that I've change the query to maxItems 50 and other parameters:
https://breno--storecomponents.myvtex.com/

#### Types of changes
